### PR TITLE
Update 1200-local-dev.md

### PR DESCRIPTION
**Pull request by sauzy34**

### CODE REVIEW

**Comment by aluzzardi** [comment]
Thank you @sauzy34 !

/cc @gerhard

File: docs/getting-started/1200-local-dev.md
Comment (TomChv) [inline docs/getting-started/1200-local-dev.md:36]: May that would be more accurate to say `or in /usr/local/bin` instead of `&`

**Review by TomChv — CHANGES_REQUESTED** [review]
Thank you for the contribution

I added a small comment to improve the sentence, feel free to reply if you think we should keep your ;)

As well, don't forget to [sign your commit](https://github.com/dagger/.github/blob/main/CONTRIBUTING.md#dco) to validate DCO. Here's a [link  to help you](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md).

File: docs/getting-started/1200-local-dev.md
Comment (helderco) [reply docs/getting-started/1200-local-dev.md:36]: I wonder why this difference in paths.

Comment (helderco) [reply docs/getting-started/1200-local-dev.md:36]: I see, it’s because of ARM and rosetta.

Comment (TomChv) [reply docs/getting-started/1200-local-dev.md:36]: That's a good question indeed, here's a [Reddit post about it](https://www.reddit.com/r/MacOS/comments/jw9guu/why_did_homebrew_move_from_usrlocalto_opthomebrew/). 

And here's the answer of Homebrew maintainer

> Another reason why we chose to support two separate installations for the time being: the native ARM one is just not ready yet so users may want to use the safer, more stable x86_64 installation (and the programs it serves) interchangeably with the newer, experimental ARM one where lots of formulae still break.

**Review by gerhard — CHANGES_REQUESTED** [review]
Nice catch @sauzy34!

There are a few changes that are needed before we can merge this:
- [ ] squash all your commits into a single one
- [ ] prefix your commit with `docs:`
- [ ] capture the actual change in your commit message, e.g. `docs: Fix brew install path for macOS ARM`
- [ ] ensure your commit keeps the `Signed off:` line (your last commit has it, but not your first one, which is making the DCO check fail

Lastly, this is the change that I think makes more sense:

```diff
$ git diff

diff --git a/docs/getting-started/1200-local-dev.md b/docs/getting-started/1200-local-dev.md
index 1423c3ba..ac38c3f0 100644
--- a/docs/getting-started/1200-local-dev.md
+++ b/docs/getting-started/1200-local-dev.md
@@ -33,13 +33,20 @@ If you do, you can install `dagger` with a single command:
 brew install dagger/tap/dagger
 \`\`\`

-This installs `dagger` in `/opt/homebrew/bin`:
+On macOS ARM this installs `dagger` in:

 \`\`\`shell
 type dagger
 dagger is /opt/homebrew/bin/dagger
 \`\`\`

+On macOS Intel this installs `dagger` in:
+
+\`\`\`shell
+type dagger
+dagger is /usr/local/bin/dagger
+\`\`\`
+
 If you do not have Homebrew installed, or you want to install a specific version of `dagger`, you can run:
```

(notice the ``` escapes above)

WDYT @sauzy34?

**Comment by slumbering** [comment]
👋 @sauzy34 ! Thanks so much for your contribution. Would it be possible to manage changes above please ? 

Have a great day :)

**Comment by gerhard** [comment]
We want to get this contribution in @sauzy34.

I have made the necessary changes in #2176 so that we can get this merged.

Thank you!

**Pull request closed without merging**
